docs/getting-started/download/java.rst

```diff
@@ -2,7 +2,13 @@
 Java
 ====
 
-.. todo:: Java installation
+The Java version of oso is available on Github. Go to the `Maven Repository <https://github.com/osohq/oso/packages/321403>`_ and download the latest jar.
+
+To use it, add it to the classpath for your project.
+
+    $ javac -classpath "oso-|release|.jar:." MyProject.java
```

Comment:
Does this actually work? I'm surprised sphinx lets you do substitutions mid-text

Reply:
Yep, 
<img width="778" alt="Screen Shot 2020-07-25 at 6 14 18 PM" src="https://user-images.githubusercontent.com/508702/88467975-0d887e80-ceab-11ea-9424-cef76d07405a.png">